tools_tag_converters/Convertor-Stubbs_Soso_to_Halo_OS_Soso.py

```diff
@@ -62,7 +60,7 @@ def __init__(self, *args, **kwargs):
         self.resizable(0, 0)
 
         self.tags_dir = StringVar(self)
-        self.tags_dir.set(curr_dir + 'tags' + PATHDIV)
+        self.tags_dir.set(curr_dir + 'tags/')
```

Comment:
@MosesofEgypt Is this trailing `/` really necessary? Also shouldn't this be using `os.path.join(curr_dir, 'tags')`.

Reply:
I'm pretty certain you can remove it from all the converters. I simply preserved the original code cause I was in a rush.